model/workflow.go

```diff
@@ -47,6 +51,20 @@ var actionsModelMapping = map[string]func(state map[string]interface{}) State{
 	StateTypeSleep:    func(map[string]interface{}) State { return &SleepState{} },
 }
 
+func init() {
+	val.GetValidator().RegisterStructValidation(continueAsStructLevelValidation, ContinueAs{})
+}
+
+func continueAsStructLevelValidation(structLevel validator.StructLevel) {
+	continueAs := structLevel.Current().Interface().(ContinueAs)
+	if len(continueAs.WorkflowExecTimeout.Duration) > 0 {
+		if err := val.ValidateISO8601TimeDuration(continueAs.WorkflowExecTimeout.Duration); err != nil {
+			structLevel.ReportError(reflect.ValueOf(continueAs.WorkflowExecTimeout.Duration),
+				"workflowExecTimeout", "dration", "reqiso8601duration", "")
+		}
```

Comment:
how about use `iso8601duration` tag？

Reply:
can be, will replace it where it occurs.